sycl/include/CL/sycl/kernel_bundle.hpp

```diff
@@ -340,7 +321,7 @@ namespace detail {
 // public onces
 __SYCL_EXPORT detail::KernelBundleImplPtr
 get_kernel_bundle_impl(const context &Ctx, const std::vector<device> &Devs,
-                       bundle_state State);
+                       bundle_state State, OSModuleHandle M);
```

Comment:
as agreed, OSModuleHandles can now be removed - conflicting kernel names in different OS modules should be  an error (according to @gmlueck). Please also add a TODO to diagnose conflicting kernel names.